browser.go

```diff
@@ -189,6 +189,14 @@ func (b *Browser) SetValue(line int) {
 	C.go_fltk_Browser_set_value((*C.Fl_Browser)(b.ptr()), C.int(line))
 }
 
+func (b *Browser) SetSelected(line int, val int) bool {
+	return int(C.go_fltk_Browser_select((*C.Fl_Browser)(b.ptr()), C.int(line), C.int(val))) != 0
+}
+
+func (b *Browser) Selected(line int) bool {
```

Comment:
Can you change the method name to IsSelected? I think it looks better for predicate functions.